specification/dimensions/adjustmenttype.md

```diff
@@ -0,0 +1,54 @@
+# adjustment type
+
+Adjustment Type indicates the record represents a change to your monthly invoice through after-the-fact adustements (e.g. refunds, pricing changes). An adjustment can be positive or negative.
+Adjustment Type can be used by practitioners to assess any impacts to spend from positive or negative adjustments.
+
+The AdjustmentType column MUST be present and MUST NOT be null or empty. This column is of type String and MUST be one of the allowed values.
+
+- Usage Correction
+- Price Correction
+- GoodWill
+- SLA Violation
+- Balance Transfer
+- General Adjustment
```

Comment:
Should we have something so generic? I worry a provider might only use this value to abide by the spec without giving practitioners what they're actually looking for.

Reply:
I think we should specify these but also allow providers to put in what they want. These are what i have seen in invoices and examples from vendors.

Reply:
Ah, so was this your suggestion instead of "Other"? If that's the case, I wonder if we should put a statement like "MUST/SHOULD NOT use "General Adjustment" unless a more specific adjustment type does not exist." (Or maybe a better wording, since not logic is difficult to follow.)

Reply:
I have updated the whole thing and used SHOULD for allowed values examples